stats/test/platform-routes.test.js

```diff
@@ -54,19 +58,7 @@ describe('Platform Routes HTTP request handler', () => {
 
   describe('GET /stations/daily', () => {
     it('returns daily station metrics for the given date range', async () => {
-      await givenDailyStationMetrics(pgPools.evaluate, '2024-01-10', [
-        { ...STATION_STATS, acceptedMeasurementCount: 1, totalMeasurementCount: 1 }
-      ])
-      await givenDailyStationMetrics(pgPools.evaluate, '2024-01-11', [
-        { ...STATION_STATS, stationId: 'station2', acceptedMeasurementCount: 1, totalMeasurementCount: 1 }
-      ])
-      await givenDailyStationMetrics(pgPools.evaluate, '2024-01-12', [
-        { ...STATION_STATS, stationId: 'station2', acceptedMeasurementCount: 2, totalMeasurementCount: 2 },
-        { ...STATION_STATS, stationId: 'station3', acceptedMeasurementCount: 1, totalMeasurementCount: 1 }
-      ])
-      await givenDailyStationMetrics(pgPools.evaluate, '2024-01-13', [
-        { ...STATION_STATS, acceptedMeasurementCount: 1, totalMeasurementCount: 1 }
-      ])
+      await givenDailyMeasurementsSummary(pgPools.evaluate)
```

Comment:
I find this test difficult to reason about because it's not clear what data is created by `givenDailyMeasurementsSummary` and how it relates to what we assert.

At the same time, we need to keep the test setup brief, so that it's clear what data is important for this test and what's just "noise" needed by our DB schema.

I propose modifying the helper to allow the caller to provide only the column values they are interested in (`day` & `station_count` in this test). The helper should backfill any missing columns with reasonable defaults.

```js
await givenDailyMeasurementsSummary(pgPools.evaluate, [
  { day: '2024-01-10', station_count: 0 },
  { day: '2024-01-11', station_count: 1 },
  { day: '2024-01-12', station_count: 2 },
  { day: '2024-01-13', station_count: 3 }
])
```


Alternatively, we can keep the concept of `STATION_STATS`, which we were using before.

I prefer the helper to backfill any missing data with reasonable defaults.